src/ui/post-cards/short-post-card/short-post-card.tsx

```diff
@@ -0,0 +1,130 @@
+import { IPostCard } from '../../../types/post-card';
+import styled from 'styled-components';
+import { ReactComponent as DotsImg } from '../../../images/dots-horizontal-svgrepo-com.svg';
+import { VotingLikeDislikeMain } from '#features/voting-up-down/voting-up-down-main/voting-up-down-main';
+import { Link } from 'react-router-dom';
+import { Bookmark } from '#features/bookmark/bookmark';
+import { showing } from '#features/dialog-image-preview/dialog-image-preview.slice';
+import { useDispatch } from 'react-redux';
+
+type PropsCard = {
+  card: IPostCard;
+};
+
+export const ShortPostcard: React.FC<PropsCard> = (props: PropsCard) => {
+  const formatDate = (dateString: string) => {
+    const options: Intl.DateTimeFormatOptions = {
+      year: 'numeric',
+      month: 'long',
+      day: 'numeric',
+    };
+    return new Date(dateString).toLocaleDateString([], options);
+  };
+
+  const dispatch = useDispatch();
+
+  return (
+    <ShortPostcardWrapper key={props.card.id}>
+      <ShortFirstLine>
+        <ShortCardFirstLine>
+          <ShortDate>{formatDate(props.card.date)}</ShortDate>
+          <Link to={`/posts/${props.card.id}`} style={{ cursor: 'pointer' }}>
```

Comment:
UI elements shouldn't have any routing logic; better to pass link as prop (preferred) or wrap the card outside to `<Link />` (this way you should be able to pass attribute "role=presentation" inside)